rb_ws/src/buggy/scripts/2d_sim/engine.py

```diff
@@ -36,9 +36,10 @@ def __init__(self, starting_pose, velocity, buggy_name):
         self.steering_subscriber = rospy.Subscriber(
             buggy_name + "/input/steering", Float64, self.update_steering_angle
         )
+        # To read from velocity 
         self.velocity_subscriber = rospy.Subscriber(
             buggy_name + "/velocity", Float64, self.update_velocity
-        )
+        ) 
```

Comment:
not a huge concern for now but watch how for trailing white space characters, they kinda clutter up the PR